src/frontend/screens/Achievements/GameAchievementDetails/index.tsx

```diff
@@ -0,0 +1,139 @@
+import { GameAchievements } from '@hyperplay/ui'
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { achievementsSortOptions } from '..'
+import walletState from 'frontend/state/WalletState'
+import { useTranslation } from 'react-i18next'
+import { observer } from 'mobx-react-lite'
+import AchievementStoreState from 'frontend/state/AchievementStoreState'
+import MintAchievementsState from 'frontend/state/MintAchievementsState'
+
+const pageSize = 6
+
+function isTimestampInPast(unixTimestamp: number) {
```

Comment:
We should move this to utliities

Reply:
sure thing. moved under helpers folder in frontend